### PR TITLE
updated tool versions

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

File: protocols/CopyToResultsDir.sh [hunk 1]
```diff
@@ -197,12 +197,10 @@ rm -f "${projectResultsDir}/rawdata/ngs/"*".phiX.recoded.fq.gz"
...
 runNumber=$(basename "$(dirname "${projectResultsDir}")")
-if [ -f "${logsDir}/${project}/${runNumber}.pipeline.started" ]
-then
-	mv "${logsDir}/${project}/${runNumber}.pipeline".{started,finished}
-else
-	touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
-fi
+
+rm -f "${logsDir}/${project}/${runNumber}.pipeline.started"
+touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
```
Comment (RoanKanninga) [inline protocols/CopyToResultsDir.sh:202]: It was so confusing to have the timestamp of the .started on the .finished file

File: templates/generate_template.sh [hunk 2]
```diff
@@ -96,10 +96,6 @@ sh "${EBROOTMOLGENISMINCOMPUTE}/molgenis_compute.sh" \
 -w "${EBROOTNGS_DNA}/create_in-house_ngs_projects_workflow.csv" \
 -rundir "${genScripts}/scripts" \
 --runid "${runID}" \
--b slurm \
---header "${EBROOTNGS_DNA}/templates/slurm/header_tnt.ftl" \
---footer "${EBROOTNGS_DNA}/templates/slurm/footer_tnt.ftl" \
---submit "${EBROOTNGS_DNA}/templates/slurm/submit.ftl" \
```
Comment (RoanKanninga) [inline templates/generate_template.sh:102]: This is not working in combination with the NGS_Automated. The scripts were not done and the automated pipeline wanted already to navigate to the jobs folder, which is not there yet.

File: protocols/MultiQC.sh [hunk 3]
```diff
@@ -55,7 +54,6 @@ echo -e "report_header_info:
 - '': '${bwaVersion}'
 - '': '${computeVersion}'
 - '': '${caddVersion}'
-- '': '${convadingVersion}'
```
Comment (RoanKanninga) [inline protocols/MultiQC.sh:58]: convading is not used anymore

File: parameters.csv [hunk 4]
```diff
@@ -35,24 +34,23 @@ gavinPlusVersion,gavin-plus/1.5.0-${javaSuffix}
...
+pythonVersion,Python/3.10.4-${toolChain_min}-bare
+rVersion,R/4.2.1-${toolChain_max}-bare
 sambambaVersion,sambamba/0.7.0
-samtoolsVersion,SAMtools/1.14-${toolChain_max}
+samtoolsVersion,SAMtools/1.14-${toolChain_min}
 seqTkVersion,seqtk/1.3-${toolChain_min}
 snpEffVersion,snpEff/4.3t-${javaSuffix}
 vcfAnnoVersion,vcfanno/v0.3.3
 vcfAnnoExecutable,vcfanno
+vcfPedVersion,VCFped/20180527_d1bbdf5-foss-2018b-Python-2.7.16
```
Comment (RoanKanninga) [inline parameters.csv:53]: hardcoded (see above)

File: protocols/CopyToResultsDir.sh (hunk 1 above)
Comment (pneerincx) [reply protocols/CopyToResultsDir.sh:202]: `rm` + `touch` is not a single atomic operation. If you have issues with the timestamp, then you can update  the timestamp by moving the touch out of the `if`:
```
if [ -f "${logsDir}/${project}/${runNumber}.pipeline.started" ]
then
	mv "${logsDir}/${project}/${runNumber}.pipeline".{started,finished}
fi
touch "${logsDir}/${project}/${runNumber}.pipeline.finished"
```

**Review by pneerincx — APPROVED** [review]

**Pull request merged**